resources/4/chartjs.js

```diff
@@ -0,0 +1,29 @@
+
+const ctx = document.getElementById('chartCanvas').getContext('2d');
+let chart = new Chart(ctx, {
+    type: 'line',
+    data: {
+        labels: ['Érték 1', 'Érték 2', 'Érték 3', 'Érték 4', 'Érték 5'],
```

Comment:
Can we query this array from the DOM instead?

Reply:
`labels: [...document.getElementsByClassName("tableHeaders")]`
```suggestion
        labels: ['Érték 1', 'Érték 2', 'Érték 3', 'Érték 4', 'Érték 5'],
```